app/helpers/medication_refill_helper.rb

```diff
@@ -3,21 +3,16 @@
 module MedicationRefillHelper
   include CalendarHelper
   # Save refill date to Google calendar
-  # rubocop:disable RescueStandardError
   def save_refill_to_google_calendar(medication)
     return true unless current_user.google_oauth2_enabled? &&
                        new_cal_refill_reminder_needed?(medication)
 
-    begin
-      args = calendar_uploader_params(medication)
-      CalendarUploader.new(args).upload_event
-    rescue
-      return_to_sign_in
-    else
-      true
-    end
+    args = calendar_uploader_params(medication)
+    CalendarUploader.new(args).upload_event
+    true
+  rescue StandardError
```

Comment:
What error are we expecting here?

Reply:
After searching around https://github.com/googleapis/google-api-ruby-client/, which is the gem we're using within `CalendarUploader`, it seems that there are two possible types of exceptions thrown: `Google::Apis::ServerError` and `Google::Apis::ClientError`.

Not entirely certain which is expected to be thrown here, so would love @julianguyen to weigh in.

Reply:
Great call out! Hmmmmm I'm not sure whether this would be a ServerError (server error) or ClientError (request invalid). It's technically possible it could be either depending on the situation. Maybe ask the folks in that repo?

https://github.com/googleapis/google-api-ruby-client/search?q=ClientError+ServerError&unscoped_q=ClientError+ServerError

Reply:
We could add both errors to cater for the two specific occasions. From my understanding, 

`Google::Apis::ServerError` 
> An error occurred on the server and the request can be retried
In short, they messed up and we can retry

`Google::Apis::ClientError`
> The request is invalid and should not be retried without modification
In short, we messed up and need to upload valid date

Reply:
Handling both sounds good to me. 😄

Reply:
One more question, why are we redirecting to `sign_in` if the upload failed? Does it mean the user is not logged in?

Reply:
That's a great question. I would assume this helper is only used where users are definitely signed in already. Will defer to @julianguyen.